packages/components/src/spinner/styles.js

```diff
@@ -0,0 +1,53 @@
+import { WRAPPER_CLASSNAME, CLASSNAME, SIZE_SMALL } from './constants';
+
+/**
+ * CSS Styles for the components.
+ *
+ * @type {string}
+ */
+export const STYLES = `
+	.${WRAPPER_CLASSNAME} {
+		background: rgba( 250, 250, 250, 0.7 );
+		bottom: 0;
+		display: none;
+		left: 0;
+		position: absolute;
+		right: 0;
+		top: 0;
+	}
```

Comment:
can we add a z-index here? Even just `2`. When applying a coupon the new PayPal button was not overlapped by the form spinner.
```suggestion
		top: 0;
		z-index: 2;
	}
```